### ISSUE TITLE
Can't access LuCi since last update: Xiaomi Router R3

### ISSUE THREAD

**Issue by Noruas29** [x-wrt/x-wrt#157]
I updated my Xiaomi Router R3 with x-wrt-9.0-b202101251550-ramips-mt7620-xiaomi_miwifi-r3-squashfs-sysupgrade.bin version... and now I can't access to my router with the web interface.

My DHCP server see it:
![freebox](https://user-images.githubusercontent.com/32742387/106376674-c61e0600-6397-11eb-82b4-63890c2c0310.png)

I can ping it:
![Ping](https://user-images.githubusercontent.com/32742387/106376682-d6ce7c00-6397-11eb-91fa-a6d8be8c640e.png)

I can access with SSH:
![ssh](https://user-images.githubusercontent.com/32742387/106376688-ea79e280-6397-11eb-9328-f61be73d2648.png)

But I can't access now with web interface:
![web](https://user-images.githubusercontent.com/32742387/106376693-fe254900-6397-11eb-85f5-8187fcce21ed.png)

I tried it with Vivaldi, Firefox, Chrome on Desktop, Laptop and Mobile, same results.

I appear with the last update, any suggestion ?

**Comment by ptpt52** [comment]
access via wan? or via lan?
default 192.168.15.1

**Comment by ptpt52** [comment]
I can confirm: no issue on my miwifi-r3

**Comment by Noruas29** [comment]
I tried both, wan and lan... same results.

**Comment by ptpt52** [comment]
try reset default config?

**Comment by Noruas29** [comment]
Before trying to reset default, I tried to install again the last firmware via sysupgrade in SSH... no problem... and I can access to LuCi.
Cool !
I updated for my language... and it crash again.
I installed again and just keep the minimal french packages:
luci-i18n-aria2-fr luci-i18n-base-fr luci-i18n-commands-fr luci-i18n-ddns-fr luci-i18n-diag-core-fr luci-i18n-firewall-fr luci-i18n-ksmbd-fr luci-i18n-mwan3-fr luci-i18n-nft-qos-fr luci-i18n-openvpn-fr luci-i18n-opkg-fr luci-i18n-p910nd-fr luci-i18n-sqm-fr luci-i18n-upnp-fr luci-i18n-wireguard-fr

My router isn't full in french, but a lots of modules are translate and LuCi is OK... I will invistigate later to understand which package crash the web interface.
Thanks for you advices.
